www/src/components/sidebar/item.js

```diff
@@ -13,76 +13,75 @@ const isItemActive = (activeItemParents, item) => {
   return false
 }
 
-class Item extends React.PureComponent {
-  constructor(props, context) {
-    super(props, context)
-    this.activeItemRef = React.createRef()
-  }
+const Item = props => {
+  const activeItemRef = useRef(false)
+  const [didScroll, setDidScroll] = useState(false)
 
-  componentDidMount() {
-    if (this.activeItemRef.current) {
-      this.activeItemRef.current.scrollIntoView({
+  useEffect(() => {
+    if (!didScroll && activeItemRef.current) {
+      activeItemRef.current.scrollIntoView({
         block: `center`,
       })
+      setDidScroll(true)
     }
-  }
+  })
```

Comment:
If there ist an empty dependency array then useEffekt will run only once
```suggestion
  },[])
```

Reply:
Then you not need `didScroll`